recipe/meta.yaml

```diff
@@ -1,34 +1,35 @@
-{% set name = "SHED" %}
-{% set version = "0.7.3" %}
-{% set sha256 = "c916c0fe726d94f6b9179511e62a673289befc1626b9c6257aad14b7ac7e288b" %}
+{% set name = "shed" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "7e051fee4be89b15c87e7b1b4994c442783b680cf28962307dbe1057d9d3ea50" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/SHED-{{ version }}.tar.gz
```

Comment:
I don't particularly like this change. Can it instead be fixed in the conda-forge feedstock, please? With this big  changes I suggest to request for review of myself or someone else from DAMA. Thanks!

Reply:
It looks like it was a plain copy of the version from CF. We don't keep the list of maintainers in this org, as everyone in the team is a maintainer. Also, having a sorted list of dependencies helps to avoid potential duplicates of the deps. I hope it makes sense.